### PR TITLE
CB-10358: Add deprecation notice

### PR DESCRIPTION
**Pull request by jcesarmobile**
### What does this PR do?
Add deprecation notice to the README

### Checklist
- [x] [Reported an issue](http://cordova.apache.org/contribute/issues.html) in the JIRA database
- [x] Commit message follows the format: "CB-3232: (android) Fix bug with resolving file paths", where CB-xxxx is the JIRA ID & "android" is the platform affected.
- [ ] Added automated test coverage as appropriate for this change.

### CODE REVIEW

**Comment by brodybits** [comment]
Should we add a statement about known security issues in the dependencies?

File: README.md [hunk 1]
```diff
@@ -16,6 +16,19 @@
...
 -->
+# cordova-weinre
+
+## Deprecation Notice
+
+This project is being deprecated. No more work will be done on this project by the Cordova development community. You can continue to use this project and it should work as-is, but issues will not be fixed by the Cordova community.
+
+weinre was built in an age when there were no remote debuggers available for mobile devices. Since then, some platforms are starting to provide remote debugger capabilities, as part of their platform tool set.
+
+1. Android:	[Get Started with Remote Debugging Android Devicese](https://developers.google.com/web/tools/chrome-devtools/remote-debugging/)
```
Comment (Menardi) [inline README.md]: Minor typo here, `Devicese` should be `Devices`

Comment (jcesarmobile) [reply README.md]: thanks!

File: README.md [hunk 2]
```diff
@@ -16,6 +16,21 @@
...
+
+This project is being deprecated. No more work will be done on this project by the Cordova development community. You can continue to use this project and it should work as-is, but issues will not be fixed by the Cordova community.
+
+weinre was built in an age when there were no remote debuggers available for mobile devices. Since then, some platforms are starting to provide remote debugger capabilities, as part of their platform tool set.
+
+1. Android:	[Get Started with Remote Debugging Android Devices](https://developers.google.com/web/tools/chrome-devtools/remote-debugging/)
+1. iOS:	[Safari Developer Tools overview](https://support.apple.com/guide/safari-developer/safari-developer-tools-overview-dev073038698/mac)
+1. Windows:	[Debug Store apps in Visual Studio](https://msdn.microsoft.com/library/hh441472.aspx)
+
+<mark>Important note</mark>, some of weinre dependencies have security issues, use on your own risk.
```
Comment (brodybits) [inline README.md]: `<mark>` doesn't seem to do much if I click the View button

I would mark "Important note" in **BOLD CAPS**

Comment (jcesarmobile) [reply README.md]: In VS Code it highlight it in yellow, but yeah, it doesn't do anything here, so I'll remove it.

**Pull request merged**